src/vsr/journal.zig

```diff
@@ -2089,7 +2110,7 @@ pub fn JournalType(comptime Replica: type, comptime Storage: type) type {
 
             // TODO The compiler should not need this align cast as the type of `headers_iops`
```

Comment:
Shall we remove this todo now that the @alignCast is removed?

Reply:
Good catch, thnks!

Reply:
fixed&rebased!